bin/testObservability/helper/helper.js

```diff
@@ -163,7 +163,7 @@ exports.getTestEnv = () => {
 }
 
 exports.getFileSeparatorData = () => {
-  const fileSeparatorRegex = /^win/.test(process.platform) ? "\\\\" : "/";
+  const fileSeparatorRegex = /^win/.test(process.platform) ? "\\" : "/";
```

Comment:
Now that `fileSeparatorRegex` & `fileSeparator` are the same values, let's refactor to send a single string and use that for all file separator operations.